.travis/after_success.sh

```diff
@@ -0,0 +1,44 @@
+#!/bin/bash
+
+# gather the coverage data
+sudo apt-get -qq install lcov
+lcov --capture --directory . -b . --output-file coverage.info
+#  filter to remove system headers
+lcov --remove coverage.info '/usr/*' -o coverage.filtered.info
+#  convert to json
+gem install coveralls-lcov
+coveralls-lcov -v -n coverage.filtered.info > coverage.c.json
+
+coverage report
+pip install coveralls-merge
+coveralls-merge coverage.c.json
+
+if [ "$TRAVIS_PYTHON_VERSION" == "2.7" ]; then
+	pip install pep8 pyflakes
+	pep8 --statistics --count PIL/*.py
+	pep8 --statistics --count Tests/*.py
+	pyflakes *.py       | tee >(wc -l)
+	pyflakes PIL/*.py   | tee >(wc -l)
+	pyflakes Tests/*.py | tee >(wc -l)
+fi
```

Comment:
So, I could have sworn that this was in the .travis.yml file that I was pulling from, but I can't find it now, and I've rebased enough that it's lost.